components/CommentForm.tsx

```diff
@@ -0,0 +1,62 @@
+import InputTextArea from './input/InputTextArea';
+import styles from '@/components/CommentForm.module.scss';
+import Button from './button/Button';
+import { ChangeEvent, FormEvent, useEffect, useState } from 'react';
+
+type TComment = {
+  content: string;
+};
+
+const initialValue: TComment = {
+  content: '',
+};
+
+function CommentForm() {
+  const [isFilled, setIsFilled] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isFilled, setIsFilled] = useState(false);
```
state의 타입이 원시타입이고 기본값이 들어간다면 타입은 생략이 가능합니다!
단, 이부분은 사람마다 취향이있는것같아요!

Reply:
disabled처리를 위해서 isFilled를 구현해주셨어요!👍
그런데 꼭 state와 effect로 처리를 해야할까요?!

commentValue자체가 state라서 상태가 변하니

 state나 effect 없이
`const isFilled = commentValue.content.trim() !== '';`
를해줘도 commentValue가 바뀔때마다 isFilled값은 바뀔 것 같아요!